src/styles/base/_globals.scss

```diff
@@ -0,0 +1,52 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  transition: all 300ms;
```

Comment:
1. it's a bad practice, please add transition to every element that can change 
2. do not use 'all', it's a good practice to set transition for each property